voyager-transitions/src/commonMain/kotlin/cafe/adriel/voyager/transitions/ScreenTransition.kt

```diff
@@ -22,14 +22,14 @@ public interface ScreenTransition {
      *
      * Returns null when it should not define a transition for this screen.
      */
-    public fun enter(): EnterTransition? = null
+    public fun enter(isPop: Boolean): EnterTransition? = null
 
     /**
      * Defines the exit transition for the Screen.
      *
      * Returns null when it should not define a transition for this screen.
      */
-    public fun exit(): ExitTransition? = null
+    public fun exit(isPop: Boolean): ExitTransition? = null
```

Comment:
I still not 100% sure if `isPop` is enough, there is cases where other events maters for transition besides pop? if so, we could pass the navigation last event directly.

Reply:
Maybe it’s better to pass the navigator itself? It will be possible to customize the animation depending on the stack (I don’t know why this might be needed)

Reply:
done